internal/pkg/image/driver/imagedriver.go

```diff
@@ -307,10 +323,11 @@ func (d *fuseappsDriver) Mount(params *image.MountParams, mfunc image.MountFunc)
 		time.Sleep(sleepTime)
 		totTime += sleepTime
 		var ws syscall.WaitStatus
-		wpid, err := syscall.Wait4(process.Pid, &ws, syscall.WNOHANG, nil)
-		if err != nil {
+		wpid, err := syscall.Wait4(cmd.Process.Pid, &ws, syscall.WNOHANG, nil)
```

Comment:
Why did this change?

Reply:
resolved in this commit
https://github.com/apptainer/apptainer/pull/1185/commits/17c45e4d5d0eab05651c9877307fcb7d58282232